static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -407,6 +414,24 @@ function SpanDetail(props: Props) {
               <Row title="Trace ID" extra={renderTraceButton()}>
                 {span.trace_id}
               </Row>
+              {flamechartSpanFeatureEnabled && profileId && event.projectSlug && (
+                <Row
+                  title="Profile ID"
```

Comment:
Odd that none of the titles are translated here. I wonder if there was a reason for it. cc @k-fish

Reply:
Probably no good one